src/frontend/js/components/PaymentButton/index.spec.tsx

```diff
@@ -168,14 +193,16 @@ describe('PaymentButton', () => {
       });
 
     render(
-      <Wrapper order={undefined} client={createTestQueryClient({ user: { username: 'John Doe' } })}>
+      <Wrapper client={createTestQueryClient({ user: true })}>
         <PaymentButton
           billingAddress={billingAddress}
           product={product}
           onSuccess={handleSuccess}
         />
       </Wrapper>,
     );
+    nbApiCalls += 1; // fetch order for useProductOrder
```

Comment:
I think we could ensure that the expected url has been called here, instead of testing counts it would be more accurate, WDYT?

Reply:
I don't think that we need to check that this precise call is made in each test.
But we want to: 
* test at least once that each call have been made at the right time.
* test that no unwanted call have been made, by checking how many queries we've at each step.

Reply:
It's up to you, I would have made sure that the URL is the expected one, as you want